src/components/BenefitsBlock/BenefitsBlock.tsx

```diff
@@ -3,64 +3,123 @@ import { BlockWrapper } from '../../ui-kit/BlockWrapper';
 import type { UniBlockProps } from '../../types';
 import { Heading } from '../../ui-kit/Heading/Heading';
 import { Img } from '../../ui-kit/Img/Img';
-import type { BenefitItemProps, BenefitsBlockContent } from './BenefitsBlockContent';
-import { DescriptionDef, ListBenefitDef, TextBenefitDef } from './BenefitsBlockContent';
+import type { BenefitsBlockContent, ListBenefitDef, TextBenefitDef } from './BenefitsBlockContent';
+import {
+  BenefitBlockItemProps,
+  BenefitsBlockVersion,
+  DescriptionDef,
+} from './BenefitsBlockContent';
+import { Picture } from '../../model/Picture';
 
 export interface BenefitsBlockProps extends BenefitsBlockContent, UniBlockProps {}
 
+const benefitsBlockStyleMap = {
+  primary: 'bg-white',
+  secondary: 'bg-primary-main',
+};
+
+const benefitsTextStyleMap = {
+  primary: '',
+  secondary: 'text-white',
+};
+
+const benefitsDescriptionStyleMap = {
+  primary: 'text-secondary-text',
+  secondary: 'text-white opacity-80',
+};
+
+const benefitsIconStyleMap = {
+  primary: 'bg-main-divider text-primary-main',
+  secondary: 'bg-white/30 text-black',
+};
+
 export const BenefitsBlock = JSX<BenefitsBlockProps>(
-  ({ className = '', title, benefitList, ...rest }) => {
+  ({ className = '', title, benefitList, benefitsBlockVersion = 'primary', ...rest }) => {
     return (
       <BlockWrapper
-        className={`font-sans text-primary-text bg-white p-12 flex flex-col items-center ${className}`}
+        className={`font-sans text-primary-text p-12 flex flex-col items-center ${className} ${benefitsBlockStyleMap[benefitsBlockVersion]}`}
         {...rest}
       >
         {title ? (
-          <Heading headingType="h3" as="h2" className="max-w-[47rem] text-center" title={title} />
+          <Heading
+            headingType="h3"
+            as="h2"
+            className={`max-w-[47rem] text-center ${benefitsTextStyleMap[benefitsBlockVersion]}`}
+            title={title}
+          />
         ) : null}
-        {benefitList?.length ? renderBenefits(benefitList) : null}
+        {benefitList?.length ? renderBenefits({ benefitList, benefitsBlockVersion }) : null}
       </BlockWrapper>
     );
   },
 );
 
-const renderBenefits = (benefits: BenefitItemProps[]) => {
+const renderBenefits = ({
+  benefitList,
+  benefitsBlockVersion,
+}: {
+  benefitList: BenefitBlockItemProps[];
+  benefitsBlockVersion: BenefitsBlockVersion;
+}) => {
```

Comment:
Тут всего два параметра. Лучше будет так:
```suggestion
const renderBenefits = (benefitList: BenefitBlockItemProps[], benefitsBlockVersion: BenefitsBlockVersion) => {
```